pages/linux/dpigs.md

```diff
@@ -0,0 +1,25 @@
+# dpigs
+
+> Show which installed packages occupy the most space.
+> Sorts the installed packages by size and outputs the largest ones.
+> More information: <https://manned.org/man/dpigs>.
+
+- Display the N largest packages on the system:
+
+`-n, --lines=N`
+
+- Use FILE instead of the default dpkg status file:
+
+`-s, --status=FILE`
+
+- Display the largest source packages of binary packages installed on the system:
+
+`-S, --source`
+
+- Display package sizes in human-readable format:
+
+`-H, --human-readable`
+
+- Display some usage information and exit:
+
+`-h, --help`
```

Comment:
This is not how our syntax works. Please take a look at other pages and see how you should format the commands.